react/Textarea/Readme.md

```diff
@@ -23,6 +23,16 @@
 </div>
 ```
 
+#### Full width textarea
+
+```
+<div>
+  <p>
+    <Textarea placeholder="I'm have a tiny size" fullwidth></Textarea>
```

Comment:
same here, let's leverage the size attribute ?

The placeholder should be changed to I'm full width